nix/t1/elaborate.nix

```diff
@@ -10,6 +10,7 @@
 , elaborator
 , config-name
 , target
+, use-binder
```

Comment:
Since the argument list is growing, I don't recommend using override in callPackage. It is better to add new argument to this function and use the currying feature to build those derivation.

For example:

```nix
{ pkgA, pkgB, pkgC }:
{ optionA, optionB, optionC}:
derivation
```

```nix
# produce a function attrset -> drv
fn = callPackage ./above_example {}; 
rtl = fn { optionA = true; ...}
```